cdap-ui/app/directives/timeline/timeline-ctrl.js

```diff
@@ -77,7 +77,7 @@ function TimelineController ($scope, LogViewerStore, LOGVIEWERSTORE_ACTIONS, myL
       names: ['system.app.log.error', 'system.app.log.warn', 'system.app.log.info', 'system.app.log.debug'],
       startTime : '',
       endTime : '',
-      resolution: '1m'
+      resolution: 'auto'
```

Comment:
Was this verified from backend? Is it ok for UI to query metrics with a resolution set to `auto`?